### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by emilyPhee**

### CODE REVIEW

File: nextjs-client/src/components/Layout/style.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import styled from '@emotion/styled';
 
 export const Container = styled.div`
-  width: 100vw;
   height: 100vh;
-  background-color: gray;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```
Comment (anthopark) [inline nextjs-client/src/components/Layout/style.js:7]: Do we need `justify-content` here?

File: nextjs-client/styles/theme.js [hunk 2]
```diff
@@ -2,19 +2,27 @@ import { extendTheme } from '@chakra-ui/react';
 
 export const theme = {
```
Comment (anthopark) [inline nextjs-client/styles/theme.js:3]: Great job organizing the various theme properties! 🚀

**Review by anthopark — APPROVED** [review]
Great job @emilyPhee. left a small question.

**Comment by anthopark** [comment]
[Footer Component](https://trello.com/c/GMfXJ0zn/3-footer-component)

File: nextjs-client/src/components/Layout/style.js (hunk 1 above)
Comment (emilyPhee) [reply nextjs-client/src/components/Layout/style.js:7]: We don't need `justify-content`. It was the default styling when I set up the Layout component.

**Pull request merged**